dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1062,6 +1062,10 @@ def generate_user(name)
   return email, password
 end
 
+def find_test_user_by_name(name)
+  User.find_by(email: @users[name][:email])
+end
```

Comment:
We've located this helper in steps.rb adjacent to the `generate_user` method which is responsible for initializing and populating the `@users` variable, and confirmed that it is accessible from pd.rb.